packages/shared_preferences/shared_preferences/extension/devtools/.gitignore

```diff
@@ -0,0 +1 @@
+build
```

Comment:
@kenzieschmoll and @stuartmorgan, I wonder if I should move this to the [root](https://github.com/flutter/packages/blob/main/.gitignore) gitignore.

The rational behind adding this here is that this is a very specific  ignore, since we need to ignore it from git but don't ignore it in `pub`.

Reply:
The root gitignore is an artifact of the fact that sometimes new files get added to the standard set of files generated by a Flutter project, and it's a pain to update 100+ example apps to account for that.

A local gitignore is my strong preference for something specific like this.